packages/components/ng-ovh-payment-method/src/components/integration/component/index.js

```diff
@@ -0,0 +1,11 @@
+import angular from 'angular';
+
+import adyen from './adyen';
+
+import component from './component';
+
+const moduleName = 'ngOvhPaymentMethodIntegrationComponent';
+
+angular.module(moduleName, [adyen]).component(component.name, component);
```

Comment:
IMO we should use a string instead of `component.name`, not easy to find where a component is declared.

Reply:
oops :)